README.md

```diff
@@ -1,204 +1,220 @@
+Console.WriteLine(enrichedActivity.ReactionCounts["like"]); // 1
+Console.WriteLine(enrichedActivity.ReactionCounts["comment"]); // 1
 
+// Fetch reactions grouped by reaction kind
+var enriched = await userFeed1.GetEnrichedFlatActivitiesAsync(GetOptions.Default.WithReaction(ReactionOption.With().Own()));
 var enrichedActivity = enriched.Results.First();
 Console.WriteLine(enrichedActivity.OwnReactions["like"]); // is the $like reaction
 Console.WriteLine(enrichedActivity.OwnReactions["comment"]); // is the $comment reaction
 
-// all reactions enrichment can be selected
-var enriched = await this._user1.GetEnrichedFlatActivities(GetOptions.Default.WithReaction(ReactionOption.With().Counts().Own().Recent()));
+// All reactions enrichment can be selected
+var enriched = await userFeed1.GetEnrichedFlatActivitiesAsync(GetOptions.Default.WithReaction(ReactionOption.With().Counts().Own().Recent()));
 
 // Personalization
-var input = new Dictionary<string, object>()
+var input = new Dictionary<string, object>
 {
     {"feed_slug", "my_personalized_feed"},
     {"user_id", "john"},
     {"limit", 20},
     {"ranking", "my_ranking"}
 };
-var response = await client.Personalization.Get("my_endpoint", input);
+var response = await client.Personalization.GetAsync("my_endpoint", input);
 
-// File & Image Uplaod
-var fileUpload = await client.Files.Upload(stream, name, contentType);
-var imageupload = await client.Images.Upload(stream, name, contentType);
-// Use fileUpload.File and imageUpload.File afterwards
+// File & Image Upload
+var fileUpload = await client.Files.UploadAsync(stream, name, contentType);
+var imageupload = await client.Images.UploadAsync(stream, name, contentType);
 
+// Use fileUpload.File and imageUpload.File afterwards
 // Open graph
-Og og = await client.Og("https://google.com");
+var og = await client.OgAsync("https://google.com");
 ```
 
-### Copyright and License Information
-
-Project is licensed under the [BSD 3-Clause](LICENSE).
-
-## Contributing
+## ✍️ Contributing
 
-We welcome code changes that improve this library or fix a problem, please make sure to follow all best practices and add tests if applicable before submitting a Pull Request on Github. We are very happy to merge your code in the official repository. Make sure to sign our [Contributor License Agreement (CLA)](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) first. See our license file for more details.
+We welcome code changes that improve this library or fix a problem, please make sure to follow all best practices and add tests if applicable before submitting a Pull Request on Github. We are very happy to merge your code in the official repository. Make sure to sign our [Contributor License Agreement (CLA)](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) first. See our [license file](./LICENSE) for more details.
```

Comment:
we should automate this via east cla or similar 

https://easycla.lfx.linuxfoundation.org/#/

cc @marco-ulge